apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -148,6 +148,16 @@ type PlacementSpec struct {
 	// +kubebuilder:default=10
 	// +kubebuilder:validation:Optional
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+
+	// StatusReportingScope controls where ClusterResourcePlacement status information is made available.
+	// When set to "ClusterScopeOnly", status is accessible only through the cluster-scoped ClusterResourcePlacement object.
+	// When set to "ClusterAndNamespaceScope", a ClusterResourcePlacementStatus object is created in the target namespace,
```

Comment:
Do we want to be clear that this only applies to CRP that selected only one namespace?